### PR TITLE
Moves logos up the page

### PR DESCRIPTION
**Pull request by GRyall**
There was a request for the egi logo to be at the top of the page (https://rt.egi.eu/rt/Ticket/Display.html?id=11546). This is a mock up of a possible way of doing it. It is more to show what it could look like than to recommend we do it. As a minimum it probably needs more refinement before merging. A screen-shot is attached. 
2016logosattop

### CODE REVIEW

File: htdocs/web_portal/components/Draw_Components/draw_page_components.php [hunk 1]
```diff
@@ -118,10 +114,17 @@ function Get_User_Info_Box_HTML()
     }
 
     /* Draws a box showing the EGI and other logos */
-    function Get_bottom_logos_Box_HTML()
+    function Get_top_logos_Box_HTML()
     {
         $HTML = "";
         $HTML .= '<div class="Left_Logo_Box left_box_menu">';
+        $HTML .= Get_File_Contents(__DIR__."/../../static_html/goc5_logo.html");
+        //Insert a portal is in read only warning message, if it is
```
Comment (tofu-rocketry) [inline htdocs/web_portal/components/Draw_Components/draw_page_components.php]: Could the "portal is in read only" bit be quoted as it's hard to parse this comment otherwise.

Comment (tofu-rocketry) [reply htdocs/web_portal/components/Draw_Components/draw_page_components.php]: Fixed it myself.

**Comment by tofu-rocketry** [comment]
What about the request for changes to the wording on the home page?

**Comment by jrha** [comment]
Whilst you're touching these, the logo images also need `alt` attributes to improve accessibility.

**Comment by GRyall** [comment]
Let#s open a seperate issue for that. I am not convinced we will use this pull request

**Comment by jrha** [comment]
Ok, done in #117.

**Comment by GRyall** [comment]
It looks unlikely we will implement this, and it will be relatively little work to reimplement if this really is required. Following discussion with @gregcorbett I am closing this pull request
